server/controllers/AuthController.js

```diff
@@ -0,0 +1,30 @@
+import Respond from '../helpers/ResponseHandler';
+import JWT from '../helpers/JWT';
+import users from '../model/users';
+
+const { jwtSign } = JWT;
+const { response } = Respond;
+class AuthController {
+  static signUp(req, res) {
+    Respond.response(res, 200, 'ok');
+  }
+
+  // eslint-disable-next-line consistent-return
+  static signIn(req, res) {
+    const { email, password: pass } = req.body;
+    const user = users.find(u => u.email === email && u.password === pass);
```

Comment:
kindly change the variable naming here, you could do `const user = users.find(user => user.email === email && user.password === password);`. it would look more readable

Reply:
I fixed it thanks